evennia/objects/tests.py

```diff
@@ -247,6 +247,51 @@ def test_get_objs_with_key_and_typeclass(self):
         )
         self.assertEqual(list(query), [self.char1])
 
+    def test_get_objs_with_key_or_alias(self):
+        query = ObjectDB.objects.get_objs_with_key_or_alias("Char")
+        self.assertEqual(list(query), [self.char1])
+        self.assertEqual(list(query), [self.char1])
+        query = ObjectDB.objects.get_objs_with_key_or_alias(
+            "Char", typeclasses="evennia.objects.objects.DefaultObject"
+        )
+        self.assertFalse(query)
```

Comment:
Should this be `self.assertNone` to be explicit?

Reply:
If we want explicit, then I'll need to check if it's equal to an empty list. I can do that. 👍